src/js/components/AddServerButton/AddServerButton.test.js

```diff
@@ -23,7 +23,10 @@ describe('AddServerButton', () => {
     const fooConfig = new Config(fooConfiguration);
 
     const wrapper = mount(
-      <GlobalContext.Provider value={{ joinRoutePath: foo, Config: fooConfig }}>
+      <GlobalContext.Provider value={{
+        joinRoutePath: foo, Config: fooConfig,
```

Comment:
```suggestion
        joinRoutePath: foo,
        Config: fooConfig,
```

If you multi-line an object, could we make a new line for each value?